ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -1,7 +1,10 @@
 import numbers
 import warnings
+from modulefinder import Module
```

Comment:
This is strange. Shouldn't this be `import torch.nn as nn` and use `nn.Module` ?